src/views/Languages/index.jsx

```diff
@@ -0,0 +1,265 @@
+import { hot } from 'react-hot-loader';
+import { Component, Fragment } from 'react';
+import Typography from '@material-ui/core/Typography';
+import { Link } from 'react-router-dom';
+import { graphql, withApollo } from 'react-apollo';
+import { withStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import Drawer from '@material-ui/core/Drawer';
+import IconButton from '@material-ui/core/IconButton';
+import Divider from '@material-ui/core/Divider';
+import Hidden from '@material-ui/core/Hidden';
+import MenuIcon from 'mdi-react/MenuIcon';
+import CloseIcon from 'mdi-react/CloseIcon';
+import { memoizeWith } from 'ramda';
+import uniqBy from 'lodash.uniqby';
+import AppBar from '../../components/AppBar';
+import TasksTable from '../../components/TasksTable';
+import Sidebar from '../../components/Sidebar';
+import ErrorPanel from '../../components/ErrorPanel';
+import Spinner from '../../components/Spinner';
+import bugsQuery from './bugs.graphql';
+import commentsQuery from './comments.graphql';
+import {
+  BUGZILLA_ORDER,
+  BUGZILLA_PAGE_NUMBER,
+  BUGZILLA_PAGE_SIZE,
+  BUGZILLA_STATUSES,
+  GOOD_FIRST_BUG,
+  BUGZILLA_LANGUAGE_MAPPING,
+} from '../../utils/constants';
+import extractWhiteboardTags from '../../utils/extractWhiteboardTags';
+
+const DRAWER_WIDTH = 300;
```

Comment:
Have this value be defined in `theme.js`. Example from tc-web: https://github.com/taskcluster/taskcluster-web/blob/df3c01703bdfc5d295c1f40988a2dbed59afbf31/src/theme.js#L86.